master/internal/db/postgres.go

```diff
@@ -804,6 +804,19 @@ WHERE state IN ('ACTIVE', 'PAUSED', 'STOPPING_CANCELED', 'STOPPING_COMPLETED', '
 	return exps, nil
 }
 
+// FailDeletingExperiment finds all experiments that were deleting when the master crashed and moves
+// them to DELETE_FAILED.
+func (db *PgDB) FailDeletingExperiment() error {
+	if _, err := db.sql.Exec(`
+UPDATE experiments
+SET state = 'DELETE_FAILED'
+WHERE state = 'DELETING'
+`); err != nil {
+		return errors.Wrap(err, "failing deleting experiments")
```

Comment:
i've moved to a small snippet describing what i was doing @ time of error like, here I was "failing experiments in the deleting state", mainly because the error messages are shorter this way (the old way i wrote these, this would've been "failed moving delete_failed experiments to delete", but it doesn't matter much to me.